apps/angularlogos/src/app/app.component.html

```diff
@@ -1,15 +1,21 @@
 <div class="main">
   <div class="content">
     <div class="intro">
-      <h1><a routerLink="/">Angular Logos</a></h1>
+      <h1>
+        <a routerLink="/">
+          <img src="/assets/angular.svg" alt="Angular Logo" />
+          Angular Logos
+        </a>
+      </h1>
       <p>
-        Welcome to Angular Logos a website dedicated to try and collect all logos, badges that are inspired on the original Angular logo.<br />
-        The logos that can be found on this website can be related to a conference, MeetUp group, NPM package or anything else that exists
+        <strong>Welcome to Angular Logos</strong> – a website dedicated to collecting all logos and badges that are inspired by the original
+        Angular logo.<br />
+        The logos that can be found on this website can be related to a conference, Meetup group, NPM package or anything else that exists
```

Comment:
After reading my own words making it plural sounds better, what do you think?

`can be related to conferences, Meetup groups, NPM packages or anything else that exists` -->  `can be related to a conference, Meetup group, NPM package`